DataFormats/CTPPSReco/src/classes_def.xml

```diff
@@ -40,6 +40,9 @@
   <ioread sourceClass="TotemRPUVPattern" version="[2]" targetClass="TotemRPUVPattern" source="bool fittable" target="fittable_">
   <![CDATA[fittable_ = onfile.fittable;]]>
   </ioread>
+  <ioread sourceClass="TotemRPUVPattern" version="[2]" targetClass="TotemRPUVPattern" source="TotemRPUVPattern::ProjectionType projection" target="projection_">
+  <![CDATA[projection_ = onfile.projection;]]>
+  </ioread>
```

Comment:
this is already in the master.
@jan-kaspar please review the commits and remove this file.
Thank you.

Reply:
Thanks for pointing this out. I will address this in the next days together with the invalid-state files.